themes/base.theme.bash

```diff
@@ -250,7 +248,9 @@ function git_prompt_vars {
 		if [[ "${SCM_GIT_GITSTATUS_RAN}" == "true" ]]; then
 			stash_count=${VCS_STATUS_STASHES}
 		else
-			stash_count="$(git stash list 2> /dev/null | wc -l | tr -d ' ')"
+			local Data
+			readarray Data <<< "$(git stash list 2> /dev/null)"
+			stash_count=${Data//[![:digit:]]/}
```

Comment:
this doesnt work as expected, it leaves the numbers in the lines, which includes the commit hash and any numbers in the last commit name...
maybe there is a way to only count the number of lines in the array? this will make more sense here

Reply:
Sorry, I think I had your use of wc(1) and tr(1) backwards. Please run this to see if it works as you intend:

```bash
readarray Data <<< "$(git stash list 2> /dev/null)"
printf '%d\n' ${#Data[@]}
```

That should work, and if so, I'll commit a fix.

Reply:
The problem is that `${#Data[@]}` returns `1` without any stash AND with one item in the stash. Maybe we should check the content of the data to see if its just an empty line or actual stash item

Reply:
Yeah, I mentioned that problem in the post to which you just replies. :P I'll address that. Sorry, I've been quite busy, lately.